testplan/testing/multitest/base.py

```diff
@@ -1158,6 +1158,9 @@ def _run_testcase(
                 ),
```

Comment:
when --code is not specified, we don't need to call report_target on testcase either. Every report_target call evokes inspect and is expensive.

Reply:
modified: report_target is now no-op when --code is not set